### DISCUSSION TITLE
Get notified by email problem

### DISCUSSION THREAD

**Discussion by VinothGox** [chaskiq/chaskiq#663] [Q&A]
Hi,

I open the chaskiq chat and first I send the message then ask the email address after entering the email still loaded not displaying any success message or displaying the email address. Please help me.  I can't solve the problem. I've attached the reference here.
![Screenshot from 2022-03-03 15-52-00](https://user-images.githubusercontent.com/76249948/156546373-723b73c1-3f92-42da-96c9-93515e53bf2b.png)

**Comment by kcalliauw** [comment]
I have the same issue. Did you ever find a solution?
